src/model/enhance-numbered-lists.ts

```diff
@@ -283,6 +330,11 @@ class Enhancer {
 		return this;
 	}
 
+	addItemLink() {
```

Comment:
```suggestion
	addItemLinks() {
```

Pluralise because it can happen multiple times